openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/HVAC1/certificate_estimation/HVAC1_PRC_calculation.py

```diff
@@ -228,3 +228,28 @@ def formula(buildings, period, parameters):
                 0, result_meet_elig
             ])
         return result_to_return
+    
+
+class HVAC1_PRC_savings_check(Variable):
+    #this variable checks if PRCs are zero, and if they are returns zero peak savings
+    value_type = float
+    entity = Building
+    definition_period = ETERNITY
+    metadata = {
+        "variable-type": "output"
+    }
+
+    def formula(buildings, period, parameters):
+        number_of_prcs = buildings('HVAC1_PRC_calculation', period)
+        peak_demand_annual_savings = buildings('HVAC1_peak_demand_annual_savings', period)
+        
+        peak_demand_savings_vs_prcs = np.select([
+                number_of_prcs <= 0,
+                number_of_prcs > 0
+        ],
+        [
+                peak_demand_annual_savings == 0,
```

Comment:
this doesn't look right....

we should do:

        peak_demand_annual_savings = np.select([
                number_of_prcs <= 0,
                number_of_prcs > 0
        ],
        [
                0,
                0]]

so essentially we need to assign value to peak_demand_annual_savings